File: server/src/routes/health.js
const { Router } = require('express');

const router = Router();

router.get('/', (req, res) => {
  res.json({
    status: 'ok',
    service: 'stocks-ai-server',
    timestamp: new Date().toISOString(),
    version: '0.1.0',
  });
});

module.exports = router;

File: ai-service/src/routes/health.py
from datetime import datetime, timezone

from fastapi import APIRouter

router = APIRouter()


@router.get("")
async def health():
    return {
        "status": "ok",
        "service": "stocks-ai-service",
        "timestamp": datetime.now(timezone.utc).isoformat(),
        "version": "0.1.0",
    }
